source/connect/in-use-encryption.txt

```diff
@@ -0,0 +1,233 @@
+.. _in-use-encryption-tab:
+
+=================
+In-Use Encryption 
+=================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. note::
+
+   In-Use Encryption is an Enterprise/Atlas only feature. 
```

Comment:
**Suggestion**

Due to the fact that we will also have a banner on this page, maybe we can make this a non-directive blurb and also mention that you need to be running a 3 node Replica set to use QE.

Reply:
Done.

Reply:
It would work with a 1 node replica set as well. A replica set is only required because QE uses transactions under the hood, and oplog exists only on the replica set.